parity-deploy.sh

```diff
@@ -318,9 +363,38 @@ if [ "$CHAIN_ENGINE" == "tendermint" ] ; then
      build_docker_config_poa
      build_docker_client
   fi	
+
+  build_spec > deployment/chain/spec.json
+  build_docker_config_ethstats
+
+elif [ "$CHAIN_ENGINE" == "kovan" ]  ; then
+
+   cp config/spec/kovan.json deployment/chain/spec.json
+
+elif [ "$CHAIN_ENGINE" == "foundation" ] ; then
+
+   cp config/spec/foundation.json deployment/chain/spec.json
```

Comment:
Can we have a quick chat on this. Will just remove these for now and we can speak about the extra chain additions and what should be static (e.g. taking kovan/foundation from parity --chain) and what can be pre-generated (e.g. multi validator set).